scripts/utils/dependencies.js

```diff
@@ -11,10 +11,10 @@ module.exports = function (web3, artifacts) {
     return Contract
   }
 
-  const BatchExchange = makeContract(require("@gnosis.pm/dex-contracts/build/contracts/BatchExchange.json"))
-  const GnosisSafe = makeContract(require("@gnosis.pm/safe-contracts/build/contracts/GnosisSafe.json"))
-  const GnosisSafeProxyFactory = makeContract(require("@gnosis.pm/safe-contracts/build/contracts/GnosisSafeProxyFactory.json"))
-  const MultiSend = makeContract(require("@gnosis.pm/safe-contracts/build/contracts/MultiSend.json"))
+  const BatchExchange = makeContract(require("../../temp/build/contracts/BatchExchange.json"))
+  const GnosisSafe = makeContract(require("../../temp/build/contracts/GnosisSafe.json"))
+  const GnosisSafeProxyFactory = makeContract(require("../../temp/build/contracts/GnosisSafeProxyFactory.json"))
+  const MultiSend = makeContract(require("../../temp/build/contracts/MultiSend.json"))
```

Comment:
Please see above comment on the networks file, we shouldn't be importing these artifacts in this way.